src/components/capture/useDiscoverCapture.ts

```diff
@@ -117,7 +120,8 @@ function useDiscoverCapture(
                     processedEntityName,
                     encryptedEndpointConfig.data,
                     options.initiateRediscovery,
-                    options.updateOnly
+                    options.updateOnly,
+                    dataPlaneName?.whole
```

Comment:
If we're only using the nested `whole` but fetching and depending on `dataPlaneName` I think there could end up being unintended side effects. We should probably just fetch `whole` directly in the hook up above.

Reply:
`dataPlaneName` was originally the complete string. When the data plane selector was developed and the name needed to be parsed, `dataPlaneName` became an object. The two workflow hooks that utilize the name did not have their dependency arrays properly updated at _that_ time. After the parsing kinks were resolved in the selector components, a commit was created that corrected this issue.

**Tl;dr:** There were commits on my local branch that were not pushed to the remote branch by the time you reviewed this PR.